.github/actions/fastlane-ios/action.yaml

```diff
@@ -0,0 +1,49 @@
+name: 'fastlane'
+description: 'Runs fastlane ci for ios platformwith target'
+inputs:
+  environment:
+    description: "According .env file provided in fastlane folder, usually one of [local, dev, staging, prod]"
```

Comment:
in your future refactoring pls implement validation for incoming inputs in a step named "validate" or "validation" so if I mistyped `environment = staing` I would get a proper message explaining my mistake.